http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/DateAccessedUpdater.scala

```diff
@@ -32,7 +32,7 @@ class DateAccessedUpdater[F[_]: ContextShift: Timer](
     (Stream.sleep[F](config.interval) ++ Stream.eval(check)).repeat
 
   private[monitor] val check: F[Unit] =
-    logger.info(s"Going to update dateAccessed") >> queue.tryDequeueChunk1(config.maxUpdate).flatMap { chunks =>
+    logger.debug(s"Going to update dateAccessed") >> queue.tryDequeueChunk1(config.maxUpdate).flatMap { chunks =>
```

Comment:
This was causing a lot of verbosity 🙈

Reply:
we can remove this line too...think I added this when I was debugging some issue